patroni/postgresql.py

```diff
@@ -151,23 +151,35 @@ def _configuration_to_save(self):
 
     @property
     def use_slots(self):
-        return self._use_slots and self._major_version >= 9.4
+        return self._use_slots and self._major_version >= 90400
 
     @property
     def callback(self):
         return self.config.get('callbacks') or {}
 
+    @staticmethod
+    def _wal_name(version):
+        return version >= 100000 and 'wal' or 'xlog'
```

Comment:
I think the more widely accepted python way to do this is with: `'xxx' if condition else 'yyy'`